save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/TestExecutionController.kt

```diff
@@ -67,15 +69,18 @@ class TestExecutionController(private val testExecutionService: TestExecutionSer
         @RequestParam page: Int,
         @RequestParam size: Int,
         @RequestParam(required = false) status: TestResultStatus?,
+        @RequestParam(required = false) testName: String?,
         @RequestParam(required = false) testSuite: String?,
```

Comment:
Pls name it to be `testSuiteName` or something more understandable then just `testSuite`

Reply:
Done